src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/exception.css.twig

```diff
@@ -1,99 +1,31 @@
-.sf-reset .traces {
-    padding: 0 0 1em 1.5em;
-}
-.sf-reset .traces a {
-    font-size: 14px;
-}
-.sf-reset .traces abbr {
-    border-bottom-color: #AAA;
-    padding-bottom: 2px;
-}
-.sf-reset .traces li {
-    color: #222;
-    font-size: 14px;
-    padding: 5px 0;
-    list-style-type: decimal;
-    margin: 0 0 0 1em;
-}
-.sf-reset .traces li.selected {
-    background: rgba(255, 255, 153, 0.5);
-}
+{{ include('@Twig/exception.css.twig') }}
```

Comment:
Yes, both functions will put the CSS code in the HTML in the end. My point is that `include()` is trying to parse the CSS file as Twig which is not needed.

It is similar to having an index.php file that contains HTML page and wants to in-line some CSS by calling `require 'theme.css.php'` in the header (where `theme.css.php` is a plain CSS file and does not contain any PHP) instead of just `echo file_get_contents('theme.css')` or `readfile('theme.css')`.

But after all it's only detail so lets not expend any more time on it. Thanks for fixing the exception page in the profiler.